packages/instant/src/util/gas_price_estimator.ts

```diff
@@ -0,0 +1,44 @@
+import { BigNumber } from '@0x/utils';
+
+import { DEFAULT_GAS_PRICE } from '../constants';
+
+interface GasStationResult {
+    average: number;
+    fastestWait: number;
+    fastWait: number;
+    fast: number;
+    safeLowWait: number;
+    blockNum: number;
+    avgWait: number;
+    block_time: number;
+    speed: number;
+    fastest: number;
+    safeLow: number;
+}
+
+const endpointUrl = 'https://ethgasstation.info/json/ethgasAPI.json';
+const fetchFastAmountInWei = async () => {
+    const res = await fetch(endpointUrl);
```

Comment:
I see that we're using `fetch` for grabbing the ETH-USD price from coinbase but by using `fetch` we're dropping support for certain browsers: https://caniuse.com/#feat=fetch. So we should either use `fetchAsync` in `@0x/utils` which uses a fetch polyfill. Or make a note and follow up, @fragosti any concerns?

Reply:
This is a good point.

Reply:
I assumed since we were using `fetch` without a polyfill for coinbase there was already an existing decision that this was OK. 
I think we either need to add a polyfill, or give an explicit deprecated warning for unsupported browsers.  I'm more in favor of the polyfill